### ISSUE TITLE
Div scrolling disabled when dragging elements in a scrollable div.

### ISSUE THREAD

**Issue by shawnwakeman** [isaacHagoel/svelte-dnd-action#583]
When you start dragging an element inside a scrollable div, the scroll event for the div is ignored. This issue only takes place when there is no scroll bar on the body, if there is, it will scroll the entire page not the div that holds the draggable elements.

**Comment by isaacHagoel** [comment]
In [this example](https://svelte.dev/repl/e2ef044af75c4b16b424b8219fb31fd9?version=3.59.2) there are a bunch of scrolling containers and it seems to work as expected. Can please you provide more details and a REPL reproducing the issue?

**Comment by shawnwakeman** [comment]
Thanks for your response! Those examples are working because the scroll bar is on the body, scrolling with an item currently being dragged, scroll events are applied to the body's scroll bar, and not the parent divs, in the REPL, start dragging an item, you will see the body's scroll bar being moved insead of the one on the parent div that the board is inside of. [REPL](https://svelte.dev/repl/28cf2063c9a1429aa64f73cb3cc422d1?version=3.59.2)

**Comment by isaacHagoel** [comment]
It is not the body that's scrolling. you can see that the headers for "doing" and "done" columns stays in place as the page is not moving.
https://github.com/isaacHagoel/svelte-dnd-action/assets/20507787/97fc00f5-d615-4e56-91eb-a346317e17ea

Same is true here:
https://github.com/isaacHagoel/svelte-dnd-action/assets/20507787/e60c7942-5885-4a63-bac1-f7fe67a39b28

**Comment by shawnwakeman** [comment]
you are right, the auto scrolling works, actually using the scroll wheel/arrow keys does not. in the original it does because the scroll events are applied to the body and there is no height cap, but in the example i provided, scrolling up and down with the wheel does not scroll through the div like it does when scrolling without dragging.

**Comment by isaacHagoel** [comment]
So if I understand correctly you are try to
1. Start dragging
2. Hover over a container
3. Use the wheel or arrow keys to make the container below scroll.
Is that correct?

**Comment by shawnwakeman** [comment]
1. Start Dragging
2. While Dragging, Use the scroll wheel 
My issue is here, the scroll is not applied properly, if you:
1. put the mouse over center(inside the div with the `class="content"`) and scroll it scrolls the internal scroll bar, and it does not do that when dragging.

**Comment by isaacHagoel** [comment]
Can you hold the mouse down and scroll with the wheel and it behaves
correctly (without dragging)?

**Comment by shawnwakeman** [comment]
yes, clicking and holding(not dragging), or just scrolling. They both scroll correctly, but when dragging it does not.

**Comment by isaacHagoel** [comment]
Interesting. The library doesn't intercept these events or anything like
that. I wonder why the behaviour is different. Not next to a computer right
now. Will try to figure it out when I have a chance

**Comment by shawnwakeman** [comment]
Thank you very much! due to some stuttering had to implement a auto scrolling system which works perfectly fine, but i'm completely stumped on the scroll behavior, the main area of issue was scroll event listers i would get an output like normal but when dragging, nothing.

**Comment by isaacHagoel** [comment]
@shawnwakeman 
I figured it out.
It happens because the lib moves the focus to the dragged element when dragged start, specifically in [this line](https://github.com/isaacHagoel/svelte-dnd-action/blob/master/src/pointerAction.js#L430).
You can try to comment it out and you'll see that it starts working.
This line was added as a fix for #135 .
If there is an alternative way to solve that issue I don't mind removing the line.
